src/components/molecules/ScheduleNG/ScheduleNG.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useMemo } from "react";
+import classNames from "classnames";
+
+import { PersonalizedVenueEvent } from "types/venues";
+
+import { eventEndTime, eventStartTime } from "utils/event";
+import { formatDateRelativeToNow, formatTimeLocalised } from "utils/time";
+
+import { Loading } from "components/molecules/Loading";
+
+import "./ScheduleNG.scss";
+
+export interface ScheduleNGProps {
+  daysEvents: PersonalizedVenueEvent[];
+  isLoading: boolean;
+}
+
+export const ScheduleNG: React.FC<ScheduleNGProps> = ({
+  daysEvents,
+  isLoading,
+}) => {
+  const hasEvents = daysEvents.length > 0;
+
+  const containerClasses = classNames("ScheduleNG");
+
+  const rowsWithTheEvents = useMemo(() => {
```

Comment:
can you use a better name like `EventsRows`?

Reply:
fixed in b61b4205dd07f566ac4c06a322bcf72fb4d174c3